tock/utilization/unit.py

```diff
@@ -49,10 +49,10 @@ def _get_unit_billing_data(users, unit, recent_periods=None, fiscal_year=False):
         start, end, utilization = utilization_report(users, unit=unit)
 
     if utilization:
-        data = utilization.values('username', 'billable', 'target')
-
+        data = utilization.values('username', 'first_name', 'last_name', 'billable', 'target')
```

Comment:
Is there still a reason to put username in `data`?

Reply:
should `username` be used as a fallback?